### PR TITLE
Update URL for new MQTT demo

### PR DESCRIPTION
**Pull request by muneebahmed10**
Description
-----------
Updates the demo URL for the new MQTT library

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/common/main.c [hunk 1]
```diff
@@ -26,7 +26,7 @@
  */
 
 /***
- * See https://www.FreeRTOS.org/mqtt/index.html for configuration and usage instructions.
+ * See https://www.FreeRTOS.org/mqtt_lts/index.html for configuration and usage instructions.
```
Comment (leegeth) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/common/main.c:29]: There are a few places  we would need to update this .url file, coupl[e of readmes I think. Can you please update those as well?

Comment (muneebahmed10) [reply FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/common/main.c:29]: Updated

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt [hunk 2]
```diff
@@ -30,6 +30,7 @@ libraries:
   + https://www.FreeRTOS.org/jobs/
   + https://www.FreeRTOS.org/ota/
   + https://www.FreeRTOS.org/pkcs11/
+  + https://www.FreeRTOS.org/mqtt_lts/
```
Comment (leegeth) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt:33]: Did we need it here? This seems to contain the demo for Beta1 libraries only

Comment (muneebahmed10) [reply FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt:33]: Good catch, I guess this what happens when the directories differ by 1 character

**Pull request merged**